script.js

```diff
@@ -16,3 +16,129 @@ menuLinks.forEach((item) => {
     menuBar.classList.toggle('open-bar');
   });
 });
+
+// details popup window
+
+const workfiles = [
+  {
+    id: 'One',
+    img: {
+      src: './Worksection Pics/dsk1.png',
+      alt: 'First Project demo',
+    },
+    title: 'Tonic',
+    type: ['CANOPY', 'Back End', '2015'],
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['HTML', 'CSS', 'Javascript'],
+  },
+  {
+    id: 'Two',
+    img: {
+      src: './Worksection Pics/dsk2.png',
+      alt: 'Second Project demo',
+    },
+    title: 'Multi-Post Stories',
+    type: ['FACEBOOK', 'Full Stack Dev', '2015'],
+    description:
+      'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    technologies: ['HTML', 'CSS', 'Javascript'],
+  },
+  {
+    id: 'Three',
+    img: {
+      src: './desktopPics/newsnap.png',
+      alt: 'Third Project demo',
+    },
+    title: 'Facebook 360',
+    type: ['CANOPY', 'BACKEND DEV', '2015'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['HTML', 'CSS', 'Javascript'],
+  },
+  {
+    id: 'Four',
+    img: {
+      src: './Worksection Pics/dsk4.png',
+      alt: 'Fourth Project demo',
+    },
+    title: 'Uber Navigation',
+    type: ['UBER', 'Lead Developer', '2018'],
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent",
+    technologies: ['HTML', 'CSS', 'Javascript'],
+  },
+];
+
+const popMain = document.querySelector('.pop-section');
+
+function popUp() {
+  workfiles.forEach((files) => {
+    popMain.innerHTML += `
+
+    <div class="pop-container" 
+    files-target="${files.id}">
+    <i class="fa fa-close" style="font-size:24px"></i>
```

Comment:
- [ ] Good job making the close button work as required. On mobile view, kindly position it correctly so that it matches the chosen Figma design. 
- [ ] Also, kindly use the correct color for the close button so that it matches the Figma design template.

**This is how I am seeing your design right now:**

![image](https://user-images.githubusercontent.com/86318284/196883352-57906fa8-887e-48d4-baec-0a96f08957f7.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/196883447-ed33fe3f-124a-4b98-80d7-f947775c5c30.png)

Reply:
Thank you for your suggestions Corrected the color and position of the Close button 👍✔